src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -41,11 +41,6 @@ protected RevitDropDownBase(string value) : base(value)
         void Controller_RevitDocumentChanged(object sender, EventArgs e)
         {
             PopulateItems();
-
```

Comment:
I believe [this issue](https://github.com/DynamoDS/Dynamo/issues/6312) was caused by this if statement.